docs/general/information-schema.rst

```diff
@@ -99,6 +99,7 @@ number of replicas.
     | pg_catalog         | pg_range                | BASE TABLE |             NULL | NULL               |
     | pg_catalog         | pg_roles                | BASE TABLE |             NULL | NULL               |
     | pg_catalog         | pg_settings             | BASE TABLE |             NULL | NULL               |
+    | pg_catalog         | pg_shdescription        | BASE TABLE |             NULL | NULL               |
```

Comment:
Jenkins failure happens as rows counts is changes, 
`SELECT 59 rows `turned into 
`SELECT 60 rows in set (... sec)`

Reply:
🙈 ... done